openflow13/instruction.go

```diff
@@ -121,7 +121,7 @@ func (instr *InstrGotoTable) AddAction(act Action, prepend bool) error {
 type InstrWriteMetadata struct {
 	InstrHeader
 	pad          []byte // 4 bytes
-	Metadata     uint64 /* Metadata value to write */
+	Metadata     uint64 /* Metadata Value to write */
```

Comment:
What is this change?

Reply:
It is a mistake generated by an IDE replace operation. Thanks for catch, I will change.